.github/workflows/deploy-website.yaml

```
@@ -27,44 +32,39 @@ permissions:
   contents: read
 
 jobs:
-  deploy:
-    name: Deploy website to S3
-    runs-on: ubuntu-latest
-    environment: ${{ inputs.environment }}
+
+  deploy_cd:
```

To be consistent with the name `manual_deploy`
```suggestion
  cd_deploy:
```